shared/constants/login.js

```diff
@@ -3,6 +3,9 @@ import * as I from 'immutable'
 import * as Types from './types/login'
 import HiddenString from '../util/hidden-string'
 
+// An ugly error message from the service that we'd like to rewrite ourselves.
+export const invalidPasswordErrorString = 'Bad password: Invalid password. Server rejected login attempt..'
```

Comment:
One period instead of two?

Reply:
it's a bug that's been there for a while now, there is some formatting which takes one message that ends with period and adds another one like `fmt.Errorf("Bad password: %s.")`